cli/src/test/integration-tests/help.sh

```diff
@@ -0,0 +1,21 @@
+echo "Running help integration test"
+output=$(ide help)
+
+assertThat() {
+  local output="$1"
+  local keyword="$2"
+
+  if echo "$output" | grep -q "$keyword"; then
+    echo "Assertion passed: '$keyword' found in output"
+    return 0
+  else
+    echo "Assertion failed: '$keyword' not found in output"
+    return 1
+  fi
+}
```

Comment:
Nice, but this is very different to what AssertJ and Java OOP can offer.
This actually implements a contains check.
Further the function returns `1` on error but the result is ignored.
You need to `exit` instead of `return` so the test actually fails on error.
This also will reveal my point if a "crashing" test will bring the entire control script down or not.

Reply:
If you want to stay with the `assertThat` pattern, you would need to do a syntax like this:
```
assertThat "$output" contains "Current version"
```
And we could implement further comparison operators later like:
```
assertThat "$versionOutput" equals "SNAPSHOT"
```
etc.

Reply:
I've adjusted the `help.sh` test now and moved the `assertThat` function to `all-tests-functions.sh`